### ISSUE TITLE
pdseg实现auxiliary loss

### ISSUE THREAD

**Issue by lyn1225** [PaddlePaddle/Paddle#27823]
语义分割领域常用auxiliary loss来优化训练，官方提供的pdseg文件loss.py中定义了multi_softmax_with_loss()函数，并且在config.py文件中也定义了cfg.MODEL.MULTI_LOSS_WEIGHT。如果要实现auxiliary loss，是否只要在相应的模型文件中返回logits的元组，并且在config.py文件中设置logits的元组所对应的MULTI_LOSS_WEIGHT就可以了？菜鸡希望大佬的解答。

**Comment by bjjwwang** [comment]
已收到，我咨询下Seg的同学回复下。

**Comment by wuyefeilin** [comment]
目前静态图没有提供支持auxiliary loss来优化训练， 里面的multi_softmax_with_loss()函数是因为icnet而设置的。 我们正在往动态图迁移，在动态图版本里是支持任意增加auxiliary设置的，如果你感兴趣的话可以尝试使用我们的动态图版本。

你目前使用的哪一个模型，目前的训练精度是多少。

有问题的话可以提到https://github.com/PaddlePaddle/PaddleSeg repo的issue里方便我们跟踪恢复。
